cineast-core/src/main/java/org/vitrivr/cineast/core/features/OCRSearch.java

```diff
@@ -25,3 +79,221 @@ protected String enrichQueryTerm(String queryTerm) {
     return queryTerm + "~1";
   }
 }
+
+class TesseractRecognizer {
+
+  public String recognizer(BufferedImage image){
+    Tesseract tesseract = new Tesseract();
+    tesseract.setDatapath("resources/OCRSearch/TesseractData/tessdata/");
```

Comment:
@ppanopticon can you upload the binary files to data.vitrivr.org so we can also fetch them as external files?